Automattic/wp-calypso
# Help Center: Underline links in article titles for better visibility

Fixes: https://github.com/Automattic/dotcom-forge/issues/10153

## Proposed Changes

* Add underline to newspack titles to improve link visibility and match /support styling.

<img width="413" alt="CleanShot 2025-01-02 at 17 30 12@2x" src="https://github.com/user-attachments/assets/685bdcf9-1811-47ff-912e-b8ef4a19609a" />


## Why are these changes being made?


* The title already look like links (blue highlight), but I added the underline to make more obvious.

## Testing Instructions



* Checkout or use live link
* Open the Help Center and open the `Themes` support doc.
* Navigate to the `Learn More About Themes` section and verify the sub titles are clickable and look like links

## Pre-merge Checklist
